mail_archives/static/description/index.html

```diff
@@ -0,0 +1,45 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Look up old mails</h2>
+            <h3 class="oe_slogan">Browse archived mails like in odoo 8</h3>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_row_img oe_centered">
+                <img class="oe_picture oe_screenshot" src="1.png"/>
+            </div>
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+                The module adds usual menu.
+            </p>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <p class="oe_mt32">
+This menu shows archive messages in threads.
```

Comment:
Actually, it doesn't shows messages in threads. It was so in 8.0 only.

     It shows all messages.